cmd/new_test.go

```diff
@@ -35,6 +36,50 @@ func newMockTime() time.Time {
 	return time.Date(2021, 5, 22, 13, 30, 10, 5, time.UTC)
 }
 
+func TestEnvOnNew(t *testing.T) {
```

Comment:
```suggestion
func TestNewWithEnvVars(t *testing.T) {
```
I try to use the command as the first part of the test.